gips/data/cdl/cdl.py

```diff
@@ -67,29 +80,48 @@ class cdlData(Data):
     _products = {
         _cdl: {
             'description': 'Crop Data Layer',
-            'assets': [_cdl],
+            'assets': [_cdl, _cdlmkii],
             'bands': [{'name': _cdl, 'units': 'none'}],
             # presently 'startdate' & 'latency' are permitted to be unspecified
             # by DH gips.utils.get_data_variables
         }
     }
 
-    _legend = None
+    def process(self, products, overwrite=False, **kwargs):
+        for asset_type, asset in self.assets.iteritems():
+            if asset_type == _cdlmkii:  # with older cdl products, the asset is the product
+                product_files = []
+
+                with ZipFile(asset.filename, 'r') as zipfile:
+                    for member in zipfile.infolist():
+                        product_files.append(zipfile.extract(member.filename))
+
+                asset_name, _ = os.path.basename(asset.filename).split('.', 1)
+
+                for f in product_files:
+                    _, ext = os.path.basename(f).split('.', 1)
+                    os.rename(f, os.path.dirname(f) + '/' + asset_name + '.' + ext)
+
+                image = GeoImage(os.path.dirname(asset.filename) + '/' + asset_name + '.tif', True)
+                image[0].SetNoData(0)
```

Comment:
If I'm reading this right and it's a conventional gips product that is distinct from its related asset file, I think you want to `self.AddFile` down here someplace; see other drivers for how they do it.

What'd be super is if the drivers didn't have so much boilerplate since they tend to work the same.